src/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+// TypeScript config file that matches all source files in the project. This file is read by 
+// IDEs and ensures that `experimentalDecorator` warnings are not showing up.
```

Comment:
While we're doing this, could you add a top-level comment to all of the tsconfig files that say what they're used for?